app/services/wallets/balance/decrease_service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Wallets
+  module Balance
+    class DecreaseService < BaseService
```

Comment:
I think this service is not used at this step, btw we also should update the `consumed_credits_amount` during this step, wdyt?

Reply:
yep fixed and moved in other PR